README.md

```diff
@@ -1,650 +1,28 @@
-[0,20] loads 21 items.
-
-```
-set.props.rangeInclusive("start","end")
-```
-
-
-1. __startIndexProperty__ <code>{String}</code>:
-  The starting property name
-1. __endIndexProperty__ <code>{String}</code>:
-  The ending property name
-
-- __returns__ <code>{[Compares](#compares-objectstringproppropavalue-bvalue-a-b-prop-algebra)}</code>:
-  Returns a prop
-
-
-### <code>set.props.enum(property, propertyValues)</code>
-
-
-Makes a prop for a set of values.
-
-```
-var compare = set.props.enum("type", ["new","accepted","pending","resolved"])
-```
-
-
-### <code>set.props.sort(prop, [sortFunc])</code>
-
-
-Defines the sortable property and behavior.
-
-```js
-var algebra = new set.Algebra(set.props.sort("sortBy"));
-algebra.index(
-  {sortBy: "name desc"},
-  [{name: "Meyer"}],
-  {name: "Adams"}) //-> 1
-
-algebra.index(
-  {sortBy: "name"},
-  [{name: "Meyer"}],
-  {name: "Adams"}) //-> 0
-```
-
-
-1. __prop__ <code>{String}</code>:
-  The sortable property.
-1. __sortFunc__ <code>{function(sortPropValue, item1, item2)}</code>:
-  The
-  sortable behavior. The default behavior assumes the sort property value
-  looks like `PROPERTY DIRECTION` (ex: `name desc`).
-
-- __returns__ <code>{[Compares](#compares-objectstringproppropavalue-bvalue-a-b-prop-algebra)}</code>:
-  Returns a compares that can be used to create
-  a `set.Algebra`.
-
-
-### <code>set.props.id(prop)</code>
-
-
-Defines the property name on items that uniquely
-identifies them. This is the default sorted property if no
-[sort](#setpropssortprop-sortfunc) is provided.
-
-```js
-var algebra = new set.Algebra(set.props.id("_id"));
-algebra.index(
-  {sortBy: "name desc"},
-  [{name: "Meyer"}],
-  {name: "Adams"}) //-> 1
-
-algebra.index(
-  {sortBy: "name"},
-  [{name: "Meyer"}],
-  {name: "Adams"}) //-> 0
-```
-
-
-1. __prop__ <code>{String}</code>:
-  The property name that defines the unique property id.
-
-- __returns__ <code>{[Compares](#compares-objectstringproppropavalue-bvalue-a-b-prop-algebra)}</code>:
-  Returns a compares that can be used to create
-  a `set.Algebra`.
-
-
-## <code>new set.Translate(clauseType, propertyName)</code>
-
-
-Localizes a clause's properties within another nested property.
-
-```js
-var algebra = new set.Algebra(
-  new set.Translate("where","$where")
-);
-algebra.has(
-  {$where: {complete: true}},
-  {id: 5, complete: true}
-) //-> true
-```
-
-This is useful when filters (which are `where` clauses) are
-within a nested object.
-
-
-1. __clause__ <code>{String}</code>:
-  A clause type.  One of `'where'`, `'order'`, `'paginate'`, `'id'`.
-1. __propertyName__ <code>{String|Object}</code>:
-  The property name which contains the clauses's properties.
+## Documentation
 
-- __returns__ <code>{[Compares](#compares-objectstringproppropavalue-bvalue-a-b-prop-algebra)}</code>:
-  A set compares object that can do the translation.
+Read the [API docs on CanJS.com](https://canjs.com/doc/can-set.html).
```

Comment:
For future runs of the codemod, it might be nice to have the package name in the link, e.g. “Read the [can-set API docs on CanJS.com](https://canjs.com/doc/can-set.html)”